### PR TITLE
Add linter testpackage

### PR DESCRIPTION
**Pull request by maratori**
Add linter [testpackage](https://github.com/maratori/testpackage)

### CODE REVIEW

File: .golangci.example.yml [hunk 1]
```diff
@@ -213,7 +213,9 @@ linters-settings:
   dogsled:
     # checks assignments with too many blank identifiers; default is 2
     max-blank-identifiers: 2
-
+  testpackage:
+    # regexp pattern to skip files
+    skip-regexp: (export|internal)_test\.go
```
Comment (ernado) [inline .golangci.example.yml:218]: Can we do same with `exlude-rules`? Seems like this option is excessive, what do you think?

File: vendor/modules.txt [hunk 2]
```diff
@@ -108,6 +108,8 @@ github.com/kisielk/gotool/internal/load
 github.com/konsorten/go-windows-terminal-sequences
 # github.com/magiconair/properties v1.8.1
 github.com/magiconair/properties
+# github.com/maratori/testpackage v1.0.0
```
Comment (ernado) [inline vendor/modules.txt:111]: Please rebase and remove vendoring :slightly_smiling_face:

File: .golangci.example.yml (hunk 1 above)
Comment (maratori) [reply .golangci.example.yml:218]: `(export|internal)_test\.go` is default skip pattern for testpackage linter. And this default is good enough for most projects. You should change it in very rare cases.

Correct me if I'm wrong. If we replace it with `exclude-rules`, we have to define default pattern for each new project explicitly.

That's why I've added this setting.

Comment (ernado) [reply .golangci.example.yml:218]: Oh, now I see, thank you.

File: vendor/modules.txt (hunk 2 above)
Comment (maratori) [reply vendor/modules.txt]: Done

**Pull request merged**